client/src/routes/[username]/[comic_slug]/[chapter_number]/+page.svelte

```diff
@@ -1,30 +1,89 @@
 <script lang="ts">
     import { page } from '$app/stores';
     import type { PageServerData } from './$types';
+    import Comment from "$lib/components/Comment.svelte";
+    import { currentUser } from '../../../stores';
+    import { goto } from '$app/navigation';
+    import type { ChapterCommentResponse } from 'bindings/ChapterCommentResponse';
 
     export let data: PageServerData;
 
-    const { chapter } = data;
+    let { chapter, comments } = data;
+
+    async function sendComment(
+        e: SubmitEvent,
+        parent_comment_id: string | null,
+        comic_id: string
+    ) {
+        const form = new FormData(e.target as HTMLFormElement);
+
+        const comment = form.get("comment");
+
+        if (comment?.toString() == null || comment?.toString().length < 1) {
+            return;
+        }
+
+        const res = await fetch(
+            `http://localhost:6060/api/v1/comics/${comic_id}/comments`,
```

Comment:
```suggestion
            `http://localhost:6060/api/v1/comics/chapters/${chapter_id}/comments`,
```

Reply:
wtf, how, it shouldn't have worked with this bug

Reply:
yeah it didn't work when I tried it

Reply:
ah, yeah, I didn't check the correctness of this file, mb